src/plugins/media-analysis/media-analysis-display.js

```diff
@@ -54,6 +62,15 @@ function MediaAnalysisDisplay({ content }) {
     setAreTextsExpanded(prevValue => !prevValue);
   };
 
+  const handleChapterClick = chapterIndex => {
+    if (playState !== MEDIA_PLAY_STATE.playing) {
+      playerRef.current.play();
+      setTimeout(() => playerRef.current.seekToPart(chapterIndex), 0);
+    } else {
+      playerRef.current.seekToPart(chapterIndex);
+    }
```

Comment:
Tiny thing: would be easier/nicer to follow the code if evaluating `playState === MEDIA_PLAY_STATE.playing` than the "not" in this case.

Reply:
True. Switched it around!